models.py

```diff
@@ -36,7 +36,7 @@ class Book(db.Model):
 
     def serialize(self):
         return {
-            'book_id': self.ourBookID,
+            'bookNumber': self.arabicBookNumber,
```

Comment:
will this be localized in future? if so, do we want to move it to block below?

Reply:
No, the book number will not be localized. It was a quirk of translations that they had different book numbers sometimes in the first place and our policy is to stick to the original work as far as possible, which will lead to a singular book number.